### PR TITLE
Add class/2 for conditional checking

### LINKED ISSUE #9: Add class/2
So we can do:

```ex
class("active", true|false)
```

This would be consistent with class_name.

### PR DESCRIPTION
**Pull request by sn3p**
Closes #9

### CODE REVIEW

File: test/ex_css_modules/ex_css_modules_test.exs [hunk 1]
```diff
@@ -93,6 +90,13 @@ defmodule ExCSSModulesTest do
       ) == "world"
     end
 
+    test "returns nil ...." do
```
Comment (jessedijkstra) [inline test/ex_css_modules/ex_css_modules_test.exs]: What's with the `....`?

Comment (sn3p) [reply test/ex_css_modules/ex_css_modules_test.exs]: woops

File: lib/ex_css_modules/ex_css_modules.ex [hunk 2]
```diff
@@ -108,12 +106,20 @@ defmodule ExCSSModules do
...
   def class_name(definition, keys) when is_list(keys) do
-    keys
-    |> Enum.map(&class_name(definition, &1))
-    |> Enum.reject(&is_nil/1)
-    |> Enum.join(" ")
+    list = keys
+      |> Enum.map(&class_name(definition, &1))
+      |> Enum.reject(&is_nil/1)
+
+    if Enum.any?(list) do
```
Comment (jessedijkstra) [inline lib/ex_css_modules/ex_css_modules.ex]: Prefer to away from if statements as much as possible, ie:
```ex
defp join_class_name(list) when length(list) == 0, do: nil
defp join_class_name(list), do: Enum.join(list, " ")
```

Comment (jessedijkstra) [reply lib/ex_css_modules/ex_css_modules.ex]: This way we can just keep piping the values as well.

File: lib/ex_css_modules/ex_css_modules.ex [hunk 3]
```diff
@@ -43,7 +43,7 @@ defmodule ExCSSModules do
   def class(definition, classes) do
     definition
     |> class_name(classes)
-    |> class_attribute
+    |> class_attribute()
```
Comment (jessedijkstra) [inline lib/ex_css_modules/ex_css_modules.ex]: :+1:

File: lib/ex_css_modules/ex_css_modules.ex (hunk 2 above)
Comment (sn3p) [reply lib/ex_css_modules/ex_css_modules.ex]: Kinda had the same thought, so you really think this is more clear?

Comment (jessedijkstra) [reply lib/ex_css_modules/ex_css_modules.ex]: I don't know yet ;) it is quite clear when you read the `join_class_name` function what it does, it gets rid of an if statement and it allows you to keep piping the value to the next function which I like a lot.

**Pull request merged**